_studio/mfx_lib/decode/mjpeg/include/mfx_mjpeg_dec_decode.h

```diff
@@ -33,6 +33,9 @@
 
 #include "umc_mutex.h"
 
+#ifdef MFX_ENABLE_SW_FALLBACK
```

Comment:
Does not make sense to protect standard header. I suggest to just include unconditionally.

Reply:
Standard headers are huge and affect compilation time. The plan was to include only required stuff.